sdk/core/core/inc/az_credentials.h

```diff
@@ -104,8 +104,9 @@ typedef struct
  * @param tenant_id an Azure tenant ID
  * @param client_id an Azure client ID
  * @param client_secret an Azure client secret
- * @return AZ_OK = Successfull initialization <br>
- * Other value = Initialization failed
+ * @retval An #az_result value indicating the result of the operation:
```

Comment:
For consistency, let's continue to use `@return`.
```suggestion
 * @return An #az_result value indicating the result of the operation:
```

See previous discussion on `@retval` vs `@return`.
https://github.com/Azure/azure-sdk-for-c/pull/622#discussion_r422412165